pkg/proto/configuration/grpc/grpc.proto

```diff
@@ -176,3 +180,17 @@ message TLSClientCertificateAuthenticationPolicy {
   // validate the remote TLS client.
   string client_certificate_authorities = 1;
 }
+
+message JWTAuthenticationPolicy {
+  // Algorithms and keys to use to authenticate the JWTs.
+  repeated JWTKeyData key = 1;
+}
+
+message JWTKeyData {
+  // Verification key in base64 format. Conflicts with key_file.
+  string key_content = 1;
+
+  // Verification key from specified path in base64 format. Conflicts with
+  // key_content.
+  string key_file = 2;
```

Comment:
Please don't add support for loading credentials from files like this. Jsonnet already provides the `importstr` keyword, which we already use to load all sorts of secrets, including TLS key material.

Reply:
I was not aware of `importstr`. Will remove the file load support.